docs/about/compatibility/docker-image-support-matrix.rst

```diff
@@ -3,7 +3,7 @@ Docker image support matrix
 ******************************************************************
 
 AMD validates and publishes `PyTorch <https://hub.docker.com/r/rocm/pytorch>`_ and
-`TensorFlow <https://hub.docker.com/r/rocm/tensorflow>`_ containers on Docker Hub. The following
+`TensorFlow <https://hub.docker.com/r/rocm/tensorflow>`_ containers on DockerHub. The following
```

Comment:
@MathiasMagnus  Docker Hub is two words, with D and H in uppercase. It's a repo name, so it will be good to follow the convention.

https://docs.docker.com/docker-hub/

@LisaDelaney Please add a guideline.

Reply:
This was an erronous attempt at pleasing the spellchecker. The change has been reverted.